backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -424,6 +430,68 @@ export class FeatureFlagService {
     return includedFeatureFlags;
   }
 
+  public async importFeatureFlags(
+    featureFlagFiles: IFeatureFlagFile[],
+    logger: UpgradeLogger
+  ): Promise<IImportError[]> {
+    logger.info({ message: 'Import feature flags' });
+    const validatedFlags = await this.validateImportFeatureFlags(featureFlagFiles, logger);
+
+    const fileStatusArray = featureFlagFiles.map((file) => {
+      const validation = validatedFlags.find((error) => error.fileName === file.fileName);
+      const isCompatible = validation && validation.compatibilityType !== FF_COMPATIBILITY_TYPE.INCOMPATIBLE;
+
+      return {
+        fileName: file.fileName,
+        error: isCompatible ? null : FF_COMPATIBILITY_TYPE.INCOMPATIBLE,
+      };
+    });
+
+    const validFiles: FeatureFlag[] = fileStatusArray
+      .filter((fileStatus) => fileStatus.error === null)
+      .map((fileStatus) => {
+        const featureFlagFile = featureFlagFiles.find((file) => file.fileName === fileStatus.fileName);
+        return JSON.parse(featureFlagFile.fileContent as string);
+      });
+
+    const createdFlags = await Promise.all(
+      validFiles.map(async (featureFlag) => {
+        const { featureFlagSegmentInclusion, featureFlagSegmentExclusion, ...rest } = featureFlag;
+        const newFlag = await this.addFeatureFlagInDB(rest as any, logger);
```

Comment:
Remove `as any` from here and make `featureFlagSegmentInclusion` and `featureFlagSegmentExclusion` optional in the types.

Reply:
code updated